config/getting-started-xmonad.hs

```diff
@@ -0,0 +1,48 @@
+import System.Exit
+import XMonad
+import XMonad.Hooks.DynamicLog
+import XMonad.Hooks.ManageDocks
+import XMonad.Hooks.ManageHelpers
+import XMonad.Layout.Fullscreen
+import XMonad.Layout.LayoutHints
+import XMonad.Layout.NoBorders
+import XMonad.Prompt
+import XMonad.Prompt.ConfirmPrompt
+import XMonad.Prompt.Shell
+import XMonad.Util.EZConfig
+import XMonad.Util.Run (hPutStrLn, spawnPipe)
+
+main = do
+  xmproc <- spawnPipe "xmobar ~/.xmonad/xmobarrc -i ~/.xmonad/icons"
+  xmonad $
+    defaultConfig
+    { terminal = "xterm"
+    , manageHook =
+        myManageHook <+> fullscreenManageHook <+> manageDocks <+>
+            manageHook defaultConfig
+    , layoutHook =
+        fullscreenFloat $ fullscreenFocus $ layoutHints $ smartBorders $
+            avoidStruts $ layoutHook defaultConfig
+    , handleEventHook = fullscreenEventHook <+> docksEventHook
+    , logHook = dynamicLogWithPP $ xmobarPP {ppOutput = hPutStrLn xmproc}
+    } `additionalKeysP`
+    [ ("M-S-q", confirmPrompt myXPConfig "exit" $ io exitSuccess)
+    , ("M-p", shellPrompt myXPConfig)
+    ] `additionalKeys`
+    [((mod4Mask, xK_q), spawn "xmonad --recompile && xmonad --restart")]
+
+myXPConfig =
+  defaultXPConfig
+  { position = Top
+  , promptBorderWidth = 0
+  , defaultText = ""
+  , alwaysHighlight = True
+  , font = "xft:Fira Mono Medium:size=9:antialias=false"
```

Comment:
Went with the even more generic `monospace` instead and also removed `antialias=false`.